STUDIO/org.openl.rules.webstudio/webapp/pages/modules/repository/modal/closeProject.xhtml

```diff
@@ -1,66 +1,83 @@
+        xmlns:rich="http://richfaces.org/rich"
+        xmlns:ui="http://java.sun.com/jsf/facelets">
 
-  <rich:popupPanel id="modalCloseProject" minWidth="530" autosized="true">
-    <f:facet name="header">
-      <h:outputText value="Confirm Close" />
-    </f:facet>
+    <rich:popupPanel id="modalCloseProject" minWidth="530" autosized="true">
+        <f:facet name="header">
+            <h:outputText value="Confirm Close"/>
+        </f:facet>
 
-    <f:facet name="controls">
-      <h:graphicImage value="/images/close.gif" class="close" onclick="RichFaces.$('modalCloseProject').hide();" alt="Close" />
-    </f:facet>
+        <f:facet name="controls">
+            <h:graphicImage value="/images/close.gif" class="close" onclick="RichFaces.$('modalCloseProject').hide();"
+                            alt="Close"/>
+        </f:facet>
 
-    <h:form id="closeProjectForm">
-      <h:panelGroup id="modalCloseProjectData">
-          <h:panelGroup rendered="#{repositoryTreeState.selectedProjectModified}">
-              <div>
-                  <h:outputText value="Project '#{node.name}' is modified, if you close it, all changes will be lost." />
-              </div>
-              <div>
-                  <h:outputText value="Are you sure you want to close the project '#{node.name}' without saving?" />
-              </div>
-          </h:panelGroup>
-        <h:panelGroup rendered="#{not repositoryTreeState.selectedProjectModified}">
-            <h:outputText value="Project '#{node.name}' will be removed from the workspace." />
-        </h:panelGroup>
-        <h:panelGroup rendered="#{repositoryTreeState.canClose and repositoryTreeController.hasDependingProjects}">
-            <div>
-                <h:outputText value="NOTE: Project " rendered="#{fn:length(repositoryTreeController.dependingProjects) == 1}"/>
-                <h:outputText value="NOTE: Projects " rendered="#{fn:length(repositoryTreeController.dependingProjects) > 1}"/>
-                <ui:repeat value="#{repositoryTreeController.dependingProjects}" var="project" varStatus="i">
-                    <h:outputText value="'#{project}'#{i.last ? '' : ', '}"/>
-                </ui:repeat>
-                <h:outputText value=" which has dependency on this project is still opened." rendered="#{fn:length(repositoryTreeController.dependingProjects) == 1}"/>
-                <h:outputText value=" which have dependency on this project are still opened." rendered="#{fn:length(repositoryTreeController.dependingProjects) > 1}"/>
-            </div>
-        </h:panelGroup>
-      </h:panelGroup>
+        <h:form id="closeProjectForm">
+            <h:panelGroup id="modalCloseProjectData">
+                    <div id="closeProjectMessagePanel" style="white-space: pre-wrap;">
+                    </div>
+            </h:panelGroup>
+            <footer>
+                <h:commandButton value="Close Project" onclick="closeProject(); return false;"

```

Comment:
Try to avoid onclick attributes in HTML. Initialise all events and listeners in JS